inst/rmarkdown/templates/tufte_book/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,264 @@
+---
+title: "Tuftish Book"
+author: "John Smith"
+date: "August 13th, 2014"
+bibliography: "knitcitations.bib"
+output: 
+  tufte_book:
+    latex_engine: lualatex
```

Comment:
If we think lualatex is preferred perhaps it should be the default? If not I don't think we should reset this here.